cookiecutter.json

```diff
@@ -8,7 +8,12 @@
   "author_email": "Enter your email (For package.json)",
   "github_org": "",
   "description": "Project Description",
-  "use_async": ["False", "True"],
+  "component_type": [
+    "Async with Class Component",
+    "Async with Function Component",
+    "Class Component",
+    "Function Component"
+  ],
```

Comment:
I don't think we need to separate them like that, I'd keep the `use_async` and add the other two option:
`Class Component` and `Function Component`.